Assignment/solution_student/index.html

```diff
@@ -10,24 +9,49 @@
 
     <link rel="shortcut icon" type="image/ico" href="images/favicon.png"><!-- the favicon -->
   </head>
+
+<header>
+          <h1>Relaxr</h1>
+          <h2>Get piece of mind with a single tap</h2>
+          <div class="nowbutton"
+          <button><a class="now" href="#">Get it Now</a></button>
```

Comment:
The `<button>` you wrapped your link in here, and around the other link at the bottom of the page is what is causing that white box ([http://d.pr/i/1hsUh](http://d.pr/i/1hsUh)), which is coming from a bunch of browser defaults for the `button` tag, but the `button` isn't really needed here, since it is probably actually just a link somewhere, so you could just remove the `<button>` element and you'll be good to go!